water_neuron/src/state/event.rs

```diff
@@ -120,6 +120,16 @@ pub enum EventType {
         #[n(1)]
         sns_proposal_id: ProposalId,
     },
+
+    #[n(16)]
+    MergeNeuron {
+        #[n(0)]
+        icp_stake_e8s: ICP,
```

Comment:
Why do you need the state, it is already in the state, same comment for the receiver.

Reply:
dfsafa

Reply:
discussed irl